.github/workflows/publish-to-ghcr-and-pypi.yml

```diff
@@ -28,10 +28,7 @@ jobs:
           python-version: "3.10"
       - name: Install pypa/build
         run: >-
-          python -m
-          pip install
-          build[uv]
-          --user
+          python -m pip install --require-hashes -r build-requirements.txt --user
```

Comment:
dumb question but how does build-requirements.txt get updated? is this a common convention?